DESCRIPTION

```diff
@@ -19,23 +19,26 @@ VignetteBuilder: knitr
 Depends:
   R (>= 2.10)
 Imports:
```

Comment:
Let's stick to 2-spaces indentation used in this files and throughout the project. It would be also nice to sort the packages alphabetically.

Reply:
Okay, I went back to using 2 spaces, packages are sorted alphabetically. 
4 spaces were introduced by `usethis::use_package` call